hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/InodeTree.java

```diff
@@ -59,9 +59,9 @@
  * {@link #resolve(String, boolean)}
  */
 
-@InterfaceAudience.Private
+@InterfaceAudience.Public
```

Comment:
Let's leave this as private for now. We really don't want to encourage other users of this class.

Reply:
@virajith, here is Owen's comment on leaving this interface as private.